app/routes/events/view/settings.js

```diff
@@ -11,5 +11,13 @@ export default Route.extend({
       roleInvites : await eventDetails.query('roleInvites', {}),
       roles       : await this.store.findAll('role')
     };
+  },
+  afterModel(model) {
```

Comment:
you need to do this in beforeModel, in afterModel the data you are trying to protect is already loaded

Reply:
@CosmicCoder96 actually there is no sensitive data shown. The Settings has only option to transfer event which this PR blocks to see for the non-admin non-owner user.
Do we still need to shift this to beforeModel?

Reply:
Actually it's better to use beforeModel bcoz no need to hit server if we don't need to. I'll shift it to beforeModel